napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -232,6 +236,9 @@ def _set_table(self, layer_str: str = ''):
                 item = QTableWidgetItem(action.description)
                 item.setFlags(Qt.ItemFlag.NoItemFlags)
                 self._table.setItem(row, self._action_name_col, item)
+                # Ensure long descriptions can be wrapped in cells
+                item.textAlignment = Qt.AlignRight | Qt.AlignVCenter
```

Comment:
```suggestion
                item.textAlignment = Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignVCenter
```

Why not use an enum explicitly?

Reply:
I think that broke with the old qt. But yea using the Enum is better for Qt6 long term?
Let's try.

Reply:
Yes.